src/Training.AirBnb.Clone.Backend/AirBnB.Api/appsettings.Development.json

```diff
@@ -36,7 +36,12 @@
   },
 
   "ConnectionStrings": {
+<<<<<<< Updated upstream
     "DbConnectionString": "Host=localhost;Port=5432;Database=Training.AirBnb.Clone;Username=postgres;Password=postgres",
+=======
+    "DefaultConnection": "Host=localhost;Port=5432;Database=Training.AirBnb.Clone;Username=postgres;Password=java2001",
+    "NotificationConnection": "Host=localhost;Port=5432;Database=Training.AirBnB.Notifications;Username=postgres;Password=java2001",
+>>>>>>> Stashed changes
```

Comment:
Remove git merging labels

Reply:
@Ilxom-22 we don't need multiple connection strings right ?

Reply:
Which connections we don't need?